lib/LaTeXML/Package/utf8.def.ltxml

```diff
@@ -29,6 +29,17 @@ AssignValue(INPUT_ENCODING => undef);
 # And set the Mouth to simply pass-thru the utf8.
 AssignValue(PERL_INPUT_ENCODING => 'UTF8');
 
+DefPrimitive('\DeclareUnicodeCharacter Digested {}', sub {
+    my ($stomach, $hexcode, $expansion) = @_;
+    my $char = $hexcode->toString();
+    if ($char =~ /^[0-9a-fA-F]+$/) {
+      $char = chr(hex($char));
```

Comment:
I just ran into [a mistake I made](https://github.com/brucemiller/LaTeXML/pull/1471/commits/09c2751436a4eb63b44ab67fb03e66079f56b426) that is also visible here - we should be using `UTF()` instead of `chr()` for these open-ended unicode codes.

I noticed it when passing resulting strings out of latexml (and out of perl) and landing on `error : string is not in UTF-8
` breakage.

Reply:
I would have never noticed this, thanks!